pkg/dao/helpers_test.go

```diff
@@ -103,11 +103,15 @@ func (s *RepositorySuite) TestUuidifyString() {
 	init_uuid, err := uuid2.NewUUID()
 	assert.NoError(t, err)
 
-	result := UuidifyString(init_uuid.String())
-	assert.Equal(t, init_uuid, result)
+	t.Run("valid uuid", func(t *testing.T) {
+		result := UuidifyString(init_uuid.String())
+		assert.Equal(t, init_uuid, result)
+	})
```

Comment:
just out of my curiosity 👀 
what's the benefit of the `t.Run()` approach? i.e. why is this needed/wanted, we haven't been using that so far 😅

Reply:
and the `TestUuidifyString` test is being added in the first commit of this PR 👀 so it probably makes to squash these 2 into one, so the commits aren't fixing/addressing each other 😅

Reply:
`t.Run()` is used for parallel run of the tests. I'm still looking at how tests are composed, but if anywhere on the higher level the `t.Parallel()` is used then these tests will use that. Also, it shows just on the code level that these are separate tests.

Yes, my general approach is to squash all commits into one during the merge, so the intermediate commits doesn't matter.

Reply:
good to know! thx 😅  
I don't think we run these in parallel, but in might make it a bit clearer that's true 👍🏼